### PR TITLE
section contact updated

### PR DESCRIPTION
**Pull request by Sjebril87**
Create spaces between inputs
Change the style of h2
Change background-image

### CODE REVIEW

File: styles/_contact.scss [hunk 1]
```diff
@@ -1,7 +1,12 @@
-
-
+h2 {
+  position: sticky;
+  top: 0;
+  transform: translateX(-50%);
+  width: 50%;
+  margin-top: 0;
+}
```
Comment (a-rfl) [inline styles/_contact.scss:7]: No need to add properties for the h2, it's already done in the general stylesheet (_basic.scss)

File: styles/_contact.scss [hunk 2]
```diff
@@ -10,19 +15,21 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding-top: 200px;
     margin: auto;
+    margin-bottom: 50px;
```
Comment (a-rfl) [inline styles/_contact.scss]: Margin too high, it's create an unwanted scroll bar.
Decrease the margin (and verifiy if it works well) **or** delete it if it's **unnecessary**.

File: styles/_contact.scss [hunk 3]
```diff
@@ -49,26 +56,27 @@
...
+    padding-top: 80px;
+  }
   .contact {
-    h2 {
-      top: 100px;
-      font-size: 40px;
-    }
     form {
-      padding-top: 180px;
+      padding-top: 60px;
```
Comment (a-rfl) [inline styles/_contact.scss:65]: May need to verifiy if it's enough for different screen size. 
Same for the next media query (for screen ->  min-height: 1000px)

File: styles/_contact.scss (hunk 2 above)
Comment (Sjebril87) [reply styles/_contact.scss]: we can delete it

**Review by a-rfl — APPROVED** [review]

**Review by kdordevic — APPROVED** [review]

**Pull request merged**